package.json

```diff
@@ -145,6 +149,12 @@
         }
       ]
     },
+    "viewsWelcome": [
+      {
+        "view": "idfSearchResults",
+        "contents": "No results found."
```

Comment:
This is supposed to be welcome message... maybe we can write here (with translation) how user can search the docs.

---

No results found can be done at the coding part where if the results array is empty add a `TreeItem` with the same title